### PR TITLE
 fix(152): cml ls --all-users display wrong owner

### PR DESCRIPTION
**Pull request by sgherdao**
alternative fix to #152 (other PR #153):

- inside `ls` command, make 1 API call to get users and create a mapping owner_id (uuid)-> username
- updated `list_lab_table` and `print_table` signatures to take the `ownerids_usernames` mapping
- add new mock for `GET /users` in `MockCMLServer`

### CODE REVIEW

File: virl/cli/ls/commands.py [hunk 1]
```diff
@@ -47,4 +49,4 @@ def ls(all, all_users):
         pl = ViewerPlugin(viewer="lab")
         pl.visualize(labs=labs, cached_labs=cached_labs)
```
Comment (xorrkaz) [inline virl/cli/ls/commands.py:50]: I think the API needs to change here as well.

Comment (xorrkaz) [reply virl/cli/ls/commands.py:50]: This would also mean a doc change.

**Review by xorrkaz — CHANGES_REQUESTED** [review]
I like the simplicity of this approach, but the plugin API needs to change.

**Comment by sgherdao** [comment]
Another way, to avoid the function signature change, we could set an attribute to the lab object:

        setattr(lab, "owner_username", ownerids_usernames[lab.owner])
 or equivalent
        lab.owner_username

but I am not sure how you feel about that

**Comment by xorrkaz** [comment]
No, I don't like this as it changes the underlying CML API.  That's more surprising for plugin developers.  I like your map approach.

**Comment by sgherdao** [comment]
Indeed, a bit hackish. I have found some bugs with the users and groups, plus plugin doc is missing. I have the changes staged. Happy to submit another Issue and/or PR or commit here.

**Pull request merged**